articles/api/management/v1/use-cases.md

```diff
@@ -8,40 +8,24 @@ contentType: reference
 useCase: invoke-api
 ---
 
-# Management API v1 Use Cases
+# Features not affected by the Management API v1 Deprecation
 
-Currently, there are API features and functionality that are only available in the [Management API v1](/api/v1). If your business process or configuration requires these features, please continue to use the API v1. Otherwise, we recommend that you use the [new version](/api/v2) instead.
+If your business process or configuration requires these features, you can continue using these endpoints as they won't be affected by the Management API v1 Deprecation.
 
-The features only available in Management API v1 include:
+The features that are not being deprecated at the moment are:
 
 * [Active Directory Connector Monitoring](#active-directory-connector-monitoring)
-* [Application Users](#application-users)
-* [Email](#email)
 * [Enterprise Users/Directory Searching](#enterprise-users/directory-searching)
-* [Rules Configuration](#rules-configuration)
-* [Searching via the PSaaS Appliance](#searching-via-the-auth0-appliance)
 
 ## Active Directory Connector Monitoring
 
-In Management API v1, there is a `GET` endpoint that allows you to monitor the status of your Active Directory Connector:
+The following endpoint allows you to monitor the status of your Active Directory Connector. It is not currently available on the [Management API v2](/api/v2).
 
 GET `/api/connections/{AUTH0_CONNECTION}/socket`
 
-## Application Users
-
-With Management API v1, after authenticating with the `client_id` and `client_secret` of an application, you can make a `GET` call to the appropriate Users endpoint to return only those users that belong to any specified application connection that is enabled for that application.
-
-[`/api/clients/{client-id}/users`](/api/v1#!#get--api-clients--client-id--users)
-
-## Email
-
-With Management API v1, you can use the `PATCH` email endpoint to update email templates as part of your automation process.
-
-[`/api/emails/{email-template-name}`](/api/v1#put--api-emails--email-template-name-)
-
 ## Enterprise Users/Directory Searching
 
-Management API v1 allows you to search directly for users authenticated using enterprise connections, such as Active Directory or Azure Active Directory.
+The following endpoints allows you to search directly for users authenticated using enterprise connections, such as Active Directory or Azure Active Directory. These endpoints are not currently available in the [Management API v2](/api/v2).
 
 * All users from a specific directory:
 [`/api/connections/{connection}/users`](/api/v1#get--api-connections--connection--users)
```

Comment:
Are we restricting which connection they can use to search?
e.g: include APIv1 enabled check inside the users search code to only allow enterprise connections (not social/database).